.prettierrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "singleQuote": true,
+  "semi": true,
+  "trailingComma": "all",
```

Comment:
Blocking: trailingComma should be none. We don't need the redundant comma at the end of array or object